tests/test_measureCrosstalk.py

```diff
@@ -0,0 +1,124 @@
+#
+# LSST Data Management System
+# Copyright 2008-2017 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+import unittest
+import numpy as np
+
+import lsst.utils.tests
+from lsst.ip.isr.measureCrosstalk import (MeasureCrosstalkTask, MeasureCrosstalkConfig)
+import lsst.ip.isr.isrMock as isrMock
+
+
+class MeasureCrosstalkTaskCases(lsst.utils.tests.TestCase):
+
+    def testMeasureCrosstalkTaskTrimmed(self):
+        """Measure crosstalk from a sequence of mocked images.
+        """
+        config = isrMock.IsrMockConfig()
+        config.rngSeed = 12345
+        config.doAddCrosstalk = True
+        config.doAddSky = True
+        config.doAddSource = True
+        config.skyLevel = 0.0
+        config.readNoise = 0.0
+        mcConfig = MeasureCrosstalkConfig()
+        mcConfig.threshold = 4000
+        mct = MeasureCrosstalkTask(config=mcConfig)
```

Comment:
This test, and the one below it, have a lot of shared setup that should probably live in a `setUp` method. Similarly for the `for` loop below. As written, I can't really tell what's different between these two tests.